int16 floor_num
---
bool result
